src/main/java/org/killbill/billing/plugin/adyen/api/mapping/UserDataMappingService.java

```diff
@@ -69,8 +73,79 @@ public static UserData toUserData(@Nullable final Account account, final Iterabl
         // set ip
         userData.setShopperIP(PluginProperties.findPluginPropertyValue(AdyenPaymentPluginApi.PROPERTY_IP, properties));
 
-        return userData;
-    }
+        //determine AccountAge Indicator
+        final String accountAgeIndicator = PluginProperties.findPluginPropertyValue(ACCOUNT_AGE_INDICATOR,properties);
+        final Optional<String> optionalAccountAgeIndicator = toLastName(accountAgeIndicator,account);
+        final String accAgeIndicator = optionalAccountAgeIndicator.isPresent() ? optionalAccountAgeIndicator.get() : null;
+        userData.setAccountAgeIndicator(accAgeIndicator);
```

Comment:
Please don't add these to `UserData` and instead create a separate `AccountInfo` class to mimic the auto-generated `AccountInfo` class in the Adyen API and make the translation from one to the other easy when creating the payment request to Adyen. This also means that this code should probably be in a new `AccountInfoMappingService` class instead.